WordPressKit/RemoteReaderPost.m

```diff
@@ -124,6 +132,13 @@ - (instancetype)initWithDictionary:(NSDictionary *)dict;
     self.isSharingEnabled = [[dict numberForKey:PostRESTKeySharingEnabled] boolValue];
     self.isLikesEnabled = [[dict numberForKey:PostRESTKeyLikesEnabled] boolValue];
 
+    // Get isWPForTeams from site options
+    NSDictionary *metaDict = [dict dictionaryForKey:PostRESTKeyMeta];
+    NSDictionary *dataDict = [metaDict dictionaryForKey:PostRESTKeyData];
+    NSDictionary *siteDict = [dataDict dictionaryForKey:PostRESTKeySite];
+    NSDictionary *optionsDict = [siteDict dictionaryForKey:PostRESTKeyOptions];
+    self.isWPForTeams = [[optionsDict numberForKey:PostRESTKeyWPForTeams] boolValue];
```

Comment:
If you want, check out the  `numberForKeyPath` method.  You could have a single const for the path. :)

Reply:
Oh. Ha! I totally didn't think of that! 🤦